src/parser/MutateSentences.h

```diff
@@ -37,49 +37,58 @@ class PropertyList final {
 
 class VertexTagItem final {
  public:
-     explicit VertexTagItem(std::string *tagName, PropertyList *properties = nullptr) {
-         tagName_.reset(tagName);
-         properties_.reset(properties);
-     }
+    explicit VertexTagItem(std::string *tagName, PropertyList *properties = nullptr) {
+        tagName_.reset(tagName);
+        properties_.reset(properties);
+    }
+
+    std::string toString() const;
 
-     std::string toString() const;
+    const std::string* tagName() const {
+        return tagName_.get();
+    }
 
-     const std::string* tagName() const {
-         return tagName_.get();
-     }
+    bool isDefaultPropNames() const {
+        return defaultPropNames_;
+    }
 
-     std::vector<std::string*> properties() const {
-         if (nullptr == properties_) {
-             return {};
-         }
-         return properties_->properties();
-     }
+    void setDefaultPropNames() {
+        defaultPropNames_ = true;
+    }
+
+    std::vector<std::string*> properties() const {
+        if (nullptr == properties_) {
+            return {};
+        }
+        return properties_->properties();
+    }
 
  private:
-     std::unique_ptr<std::string>               tagName_;
-     std::unique_ptr<PropertyList>              properties_;
+    bool                                       defaultPropNames_{false};
+    std::unique_ptr<std::string>               tagName_;
+    std::unique_ptr<PropertyList>              properties_;
 };
 
 
 class VertexTagList final {
  public:
```

Comment:
need to delete the space before `public`

Reply:
done